#!/usr/bin/env python
PACKAGE = "rosplane"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# Chi Infinity
gen.add("CHI_INFTY", double_t, 0, "Chi Infinity", 1.5, 0 , 1.5708)

# K Path
gen.add("K_PATH", double_t, 0, "K Path", 0.065, 0, 1)

# K Orbit
gen.add("K_ORBIT", double_t, 0, "K Orbit", 6.5, 0, 15)

exit(gen.generate(PACKAGE, "rosplane", "Follower"))
